docs/documentation/after-installation-guide.md

```diff
@@ -0,0 +1,243 @@
+  ```
+* Next, update the repositories:
+  ```
+  sudo dnf update
+  ```
+* Install the AMD drivers package:
+  ```
+  sudo dnf install akmod-amdgpu
+  ```
+* Reboot your system to load the driver.
+
+## Configure firewall settings
+
+To configure firewall settings on AlmaLinux, you can use the built-in firewall management tool - **firewalld**.
+* Install the tool:
+  ```
+  sudo dnf install firewalld
+  ```
+* Start the firewalld service:
+  ```
+  sudo systemctl start firewalld
+  ```  
+* If needed, enable it to start automatically on boot:
+  ```
+  sudo systemctl enable firewalld
+  ```
+* Use the `firewall-cmd` command to define rules and zones for your firewall.
+
+[Read more](https://wiki.almalinux.org/series/system/SystemSeriesA02.html#%F0%9F%93%96-interacting-with-firewalld) about the tool's usage.
+
+## Desktop Environment
+
+One of the possible customizations to your AlmaLinux system is having a Desktop Environment of your choice. AlmaLinux supports:
+* GNOME
+* GNOME-Mini
+* KDE
+* XFCE
+* MATE
+* Pantheon for AlmaLinux 9
+
+Some AlmaLinux images already have GNOME and GNOME-Mini desktop environments installed. 
+
+To install GNOME, KDE, and XFCE desktop environments:
+* First, enable EPEL and PowerTools/CRB repositories.
+* List available available package groups:
+  ```
+  sudo dnf group list
+  ``` 
+  An example output you may see:
+  ``` 
+  Available Environment Groups:
+    Server with GUI # GNOME Desktop Environment
+    Server 
+    Workstation # GNOME Desktop Environment
+    KDE Plasma Workspaces # KDE Desktop Environment
+    Virtualization Host
+    Custom Operating System
+    ...
+  Available Groups:
+    ...
+    Xfce # XFCE Desktop Environment
+  ```
+* Install a package group of your choice:
+    * GNOME - you can install one of the "Server with GUI" or "Workstation" options to get the GNOME Desktop Environment. Check the list of packages to define which option fits your needs better:
+      ```
+      dnf group info "Server with GUI" # to list packages of Server with GUI group
+      dnf group info "Workstation" # to list packages of Workstation group
+      ```
+      Install the preferred one:
+      ```
+      sudo dnf groupinstall "Server with GUI" # install Server with GUI group of packages
+      sudo dnf groupinstall "Workstation" # install Workstation group of packages
+      ```
+    * KDE
+      ```
+      sudo dnf groupinstall "KDE Plasma Workspaces" 
+      ```
+    * XFCE
+      ```
+      sudo dnf groupinstall "Xfce"
+      ```
+
+To install Pantheon for AlmaLinux 9:
+* Enable EPEL, PowerTools and Synergy repositories
+* Install Pantheon Desktop Environment:
+  ```
+  sudo dnf groupinstall "Pantheon Desktop"
+  ```
+
+To have an AlmaLinux system with GNOME-Mini or MATE desktop environments we recommend installing AlmaLinux OS from the corresponding Live Media image.
```

Comment:
Should we link to the live media page here?

Reply:
@bennyvasquez linked and fixed a bit other links